notifications_utils/recipients.py

```diff
@@ -366,47 +369,63 @@ class InvalidAddressError(InvalidEmailError):
     pass
 
 
-def validate_phone_number(number, column=None):
+def normalise_phone_number(number):
```

Comment:
Is it worth having a test for this, given that it does raise an exception and is used by the validation functions?